README.md

```diff
@@ -15,6 +15,49 @@ def deps do
 end
 ```
 
+## Usage
+
+If the data is valid, the function returns {:ok, data}. The data returned will be coerced according to the schema defined.
+
+If the data passed does not follow the rules defined in the schema, the function returns {:error, error_message}. It will also return error, in case we have a field in data, whose rule is not defined in the schema.
+
+```elixir
+iex> schema = %{"id": %Litmus.Type.Any{"required": true}}
+iex> params = %{"id": 1}
```

Comment:
same here, we don't need the quotes around `id`.

Reply:
i think this actually should be `"id" =>` to make it a string key not an atom.